.gitignore

```diff
@@ -0,0 +1,9 @@
+.envrc
+*.tar.gz
+tf.json
+*terraform.tfstate*
+.terraform
+.terraform.lock.hcl
+terraform.tfvars
+*kubeconfig
+!*kubeconfig/
```

Comment:
Add newline at the end

Reply:
.gitignore shouldn't have a trailing newline but I've updated it anyways.